c7n/resources/elasticache.py

```diff
@@ -33,7 +35,10 @@
 
 filters = FilterRegistry('elasticache.filters')
 actions = ActionRegistry('elasticache.actions')
+#registered marked-for-op filter
+filters.register('marked-for-op', tags.TagActionFilter) 
 
+ttype = re.compile('cache.t')
```

Comment:
generally for global variables, that are referenced from inside functions, best practice is to UPPER_CASE them so they stand out. 

else looks good, thanks